tests/test_views.py

```diff
@@ -278,6 +284,49 @@ def test_collections_s2_radiometry(self, api):
                 assert [b[f] for b in resp['summaries']['eo:bands'] if f in b] == [b[f] for b in eo_bands if f in b]
 
 
+TEST_AWS_REGION_NAME = 'eu-central-1'
+
+@pytest.fixture(scope='function')
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ['AWS_ACCESS_KEY_ID'] = 'testing'
+    os.environ['AWS_SECRET_ACCESS_KEY'] = 'testing'
+    os.environ['AWS_SECURITY_TOKEN'] = 'testing'
+    os.environ['AWS_SESSION_TOKEN'] = 'testing'
+    os.environ['AWS_DEFAULT_REGION'] = TEST_AWS_REGION_NAME
+    os.environ['AWS_REGION'] = TEST_AWS_REGION_NAME
```

Comment:
this fixture does not reset on exit. Can we use that monckeypath thing here too?